Refresh.GameServer/Types/Reviews/RatingType.cs

```diff
@@ -5,4 +5,22 @@ public enum RatingType : sbyte
     Yay = 1,
     Neutral = 0,
     Boo = -1,
+}
+
+public static class RatingTypeExtensions
+{
+    public static int ToDPad(this RatingType type)
+    {
+        return (int)type;
+    }
+
```

Comment:
Not sure what the point of this method is?

Reply:
Mostly visual clarity when reading code, its not needed at all

Reply:
Feel free to keep it in that case